covid_model/regional_model.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
```

Comment:
this model adds a region compartment, and is able to produce contact matrices for a set of regions.